FILES/GetDlcVehicleData.md

```diff
@@ -10,12 +10,15 @@ BOOL GET_DLC_VEHICLE_DATA(int dlcVehicleIndex, int* outData);
 
 ```
 dlcVehicleIndex takes a number from 0 - GET_NUM_DLC_VEHICLES() - 1.  
-outData is a struct of 3 8-byte items.  
-The Second item in the struct *(Hash *)(outData + 1) is the vehicle hash.  
+outData is a struct of 3 8-byte items.
+retval is a boolean value return if find or not the dlcVehicleIndex.
+The Second item in the struct *(Hash *)(outData + 1) is the vehicle hash.
 ```
 
 ## Parameters
 * **dlcVehicleIndex**: 
 * **outData**: 
 
 ## Return value
+* **retval**
```

Comment:
There should only be one return value and it should be a description, not a list like the parameters. For an example return value description, look [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md).